Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -162,6 +162,18 @@ import Foundation
             }
         }
     }
+    
+    func fetchConfiguration() async throws -> BTConfiguration {
```

Comment:
Added because this makes life a lot easier within the `BTAnalyticsService.performEventRequest` method for testing

Reply:
Would love to move all `fetchOrReturnRemoteConfig` calls to this syntax at some point